exonum-java-binding-common/pom.xml

```diff
@@ -55,10 +57,35 @@
     <dependency>
       <groupId>com.google.auto.value</groupId>
       <artifactId>auto-value</artifactId>
-      <version>1.6.2</version>
+      <version>${auto-value.version}</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.ryanharter.auto.value</groupId>
+      <artifactId>auto-value-gson-annotations</artifactId>
+      <version>${auto-value-gson.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.ryanharter.auto.value</groupId>
+      <artifactId>auto-value-gson</artifactId>
+      <version>${auto-value-gson.version}</version>
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>${gson.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.squareup</groupId>
+      <artifactId>javapoet</artifactId>
+      <version>${javapoet.version}</version>
```

Comment:
That is weird that this dependency is required: I ran ejb-common tests on JDK 8 & 11 locally without this explicit transitive dependency, and they succeed: `mvn test -pl exonum-java-binding-common/ -am`

Which tests fail and in which environment?

Reply:
I can check, maybe it was essential whrn I use @TypeAdapter annotations.